README.md

```diff
@@ -1 +1,153 @@
-# Raj-linters
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <img src="logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Microverse README Template</b></h3>
+</div>
+# 📗 Table of Contents
+
+- [📖 About the project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+**[Hello Microverse Project]** is a Hello Microverse git hub repository without any complex coding set up with proper linter of HTML and CSS which also includes new branch that is created in accordance with the GitHub flow rules in order to make code concise,correct and standardised.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The core concepts of HTML,CSS and git hub are implemented.These three tools are essential for modern web development and play a critical role in building high-quality websites and applications.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```

Comment:
Nice section here 💯 but make sure that it contains information relevant to your project and the technologies you used. 😉